e2e_playwright/st_text_test.py

```diff
@@ -22,26 +22,33 @@ def test_st_text_shows_correct_text(app: Page):
     expect(app.get_by_test_id("stText").nth(0)).to_have_text("This text is awesome!")
 
 
-def test_st_text_shows_ascii_art_correctly(
-    app: Page, assert_snapshot: ImageCompareFunction
-):
-    assert_snapshot(app.get_by_test_id("stText").nth(1), name="st_text-ascii_art")
-
-
 def test_st_text_doesnt_apply_formatting(
     app: Page, assert_snapshot: ImageCompareFunction
 ):
     assert_snapshot(
-        app.get_by_test_id("stText").nth(2), name="st_text-no_formatting_applied"
+        app.get_by_test_id("stText").nth(1), name="st_text-no_formatting_applied"
     )
 
 
 def test_help_tooltip_works(app: Page):
     """Test that the help tooltip is displayed on hover."""
-    text_with_help = app.get_by_test_id("stText").nth(3)
+    text_with_help = app.get_by_test_id("stText").nth(2)
     expect_help_tooltip(app, text_with_help, "This is a help tooltip!")
 
 
+def test_multiline_text(app: Page):
+    """Test that multiline text is displayed correctly."""
+    multiline_text = app.get_by_test_id("stText").nth(3).locator("div")
+    bounding_box = multiline_text.bounding_box()
+    assert bounding_box["height"] > bounding_box["width"]
```

Comment:
@raethlein I didn't find a good way to validate if we actually render newlines here. Suggestions were like to look for line break characters.
I ended up checking the dimensions here, given that our `st.text` is standardized in font family and size. (A future edge case would be that we use some ultra-wide font for `st.text`)

Reply:
I like the idea 🙂 An alternative way might be to check whether there is an actual character sequence `\n` in the text itself. Though if this is not possible or too tricky, it should be fine to leave a comment to this approach and move on 👍

Reply:
Added
1. `expect(multiline_text).not_to_contain_text("\\n")` to multi-line test to ensure we don't actually display the `\n` characters
2. singe line test *with* literal `\n` characters and ensure it doesn't do line break, i.e. `expect(singleline_text).not_to_contain_text(repr("\n"))` being true